network.go

```diff
@@ -390,6 +390,50 @@ func createNetworkEndpoints(numOfEndpoints int) (endpoints []Endpoint, err error
 	return endpoints, nil
 }
 
+func getIfacesFromNetNs(networkNSPath string) ([]net.Interface, error) {
+	var ifaces []net.Interface
+	var err error
+
+	if networkNSPath == "" {
+		return []net.Interface{}, fmt.Errorf("Network namespace path cannot be empty")
+	}
+
+	err = doNetNS(networkNSPath, func(_ ns.NetNS) error {
+		ifaces, err = net.Interfaces()
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+	if err != nil {
+		return []net.Interface{}, err
+	}
+
+	return ifaces, err
+}
+
+func getIPNetFromEndpoint(endpoint Endpoint, netIfaces []net.Interface) (net.IPNet, error) {
```

Comment:
Oh no wait, we can do that, just adding each ipAddress found for the same interface index.